src/robot/model/visitor.py

```diff
@@ -490,6 +491,29 @@ def end_break(self, break_: Break):
         """
         self.end_body_item(break_)
 
+    def visit_error(self, error: Error):
+        """Visits ERROR elements."""
```

Comment:
Doc like this would be better:
```python
"""Visits body items resulting from invalid syntax.

Examples include syntax like ``END`` or ``ELSE`` in wrong place and
invalid setting like ``[Invalid]``.
"""
```
Earlier docs use style like `BREAK` because such syntax actually exists, but we don't have any explicit `ERROR` syntax.